invenio_circulation/api.py

```diff
@@ -34,8 +34,9 @@ def create(cls, data, id_=None, **kwargs):
         """Create Loan record."""
         data["$schema"] = current_jsonschemas.path_to_url(cls._schema)
         ref_builder = current_app.config.get("CIRCULATION_ITEM_REF_BUILDER")
-        item_attached = data.get("item_pid") and data["item_pid"] != ""
-        if ref_builder and item_attached:
+        item_pid = data.get("item_pid")
+        if ref_builder and item_pid:
+            data["document_pid"] = get_document_by_item_pid(item_pid)
```

Comment:
Maybe `get_document_by_item_pid` should be renamed to `get_document_pid_by_item_pid`. What do you think?

Reply:
Well obviously since it returns the `document_pid`, nice catch ;)